src/Krank/Checkers/IssueTracker.hs

```diff
@@ -138,7 +139,13 @@ headersFor issue = do
 httpExcHandler :: Localized GitIssue
                -> Req.HttpException
                -> ReaderT KrankConfig IO Value
-httpExcHandler issue _ = pure . AesonT.object $ [("error", AesonT.String . pack . show $ issueUrl . unLocalized $ issue)]
+httpExcHandler issue exc = pure . AesonT.object $ [("error", AesonT.String . pack $
+  [fmt|
+    Error:
+      {(showHTTPException exc)}
+    Requested URL:
+      {(show $ issueUrl . unLocalized $ issue)}
```

Comment:
Suggestion: remove the `Requested URL` here, because it is already formatted by the top level formater, resulting in the url being printed two times.